spec/support/shared_examples/models/shared_examples_for_base_claim.rb

```diff
@@ -44,4 +44,18 @@
     it { is_expected.to accept_nested_attributes_for(:assessment) }
     it { is_expected.to accept_nested_attributes_for(:redeterminations) }
   end
+
+  describe '#cleaner' do
+    subject(:claim) { create(:draft_claim) }
```

Comment:
this creates an advocate final claim so is not a shared example for any claim type. The subject should be set by the calling spec.

following this refactors logic a base claim is just expected to respond to #cleaner, and claim types expected to call a specific cleaner.

so,  remove subject and, optionally, add
```
  it { is_expected.to respond_to(:cleaner) }
```